components/BookshelfItem.vue

```diff
@@ -69,7 +69,9 @@
       <NuxtLink
         :to="bookInfo.authorPageRoute.value"
         class="inline-block mt-0.5 text-xs laptop:text-sm text-[#9B9B9B] hover:text-theme-500 line-clamp-1 hover:underline"
-      >{{ bookInfo.authorName }}</NuxtLink>
+      >
+        {{
+          bookInfo.authorName }}</NuxtLink>
```

Comment:
Why the strange new line?

```suggestion
      >
        {{ bookInfo.authorName }}
      </NuxtLink>
```

Reply:
Updated